core/bin/oauth.rs

```diff
@@ -243,6 +306,14 @@ fn main() {
                 "/connections/:connection_id/access_token",
                 post(connections_access_token),
             )
+            .route(
+                "/oauth-service/credentials",
```

Comment:
why `oauth-service` this is inconsistent with the existing endpoitns?

Reply:
should just be `/credentials` I think. Although I am still most bullish on calling the resource `secrets`

Reply:
oauth-service has probably been introduced by copilot, i overlooked it :)